Sources/EpoxyCollectionView/ViewControllers/CollectionViewController.swift

```diff
@@ -18,11 +18,13 @@ open class CollectionViewController: UIViewController {
   public init(
     layout: UICollectionViewLayout,
     sections: [SectionModel]? = nil,
-    configuration: CollectionViewConfiguration = .shared)
+    configuration: CollectionViewConfiguration = .shared,
+    usesSafeAreaLayoutGuideLeadingTrailing: Bool = false)
```

Comment:
is there a reason we have this as a separate param instead of adding it to `CollectionViewConfiguration`?

Reply:
no, I was prototyping and I didn't look into the configuration object before opening the pr 😬